Can't compile

Linking error. I know I need the XCB library installed, but not sure which that is on Ubuntu. It looks like I have libxcb1 installed and libxcb1-dev
```
❯ cargo build --release
   Compiling libc v0.2.69
   Compiling memchr v2.3.3
   Compiling log v0.4.8
   Compiling lazy_static v1.4.0
   Compiling cfg-if v0.1.10
   Compiling regex-syntax v0.6.17
   Compiling thread_local v1.0.1
   Compiling aho-corasick v0.7.10
   Compiling xcb v0.9.0
   Compiling regex v1.3.7
   Compiling umberwm v0.0.17
   Compiling myumberwm v0.0.1 (/home/dalton/dev/myumberwm)
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-m64" "-Wl,--eh-frame-hdr" "-L" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.0.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.1.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.10.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.11.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.12.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.13.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.14.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.15.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.2.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.3.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.4.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.5.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.6.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.7.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.8.rcgu.o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.myumberwm.4jd5ph17-cgu.9.rcgu.o" "-o" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785" "/home/dalton/dev/myumberwm/target/release/deps/myumberwm-543f261b8bd8e785.275em5cn4vgv6nhk.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-L" "/home/dalton/dev/myumberwm/target/release/deps" "-L" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/dalton/dev/myumberwm/target/release/deps/libumberwm-d5eb8012ff3466a6.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libregex-10c524ff798ebf91.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libthread_local-3948ce50dadeb546.rlib" "/home/dalton/dev/myumberwm/target/release/deps/liblazy_static-72fd4c50b9d3882e.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libregex_syntax-420be60029984cc8.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libaho_corasick-4c484a23c2e706ee.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libmemchr-b5ad103136b5cebd.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libxcb-a8458a241519b5ad.rlib" "/home/dalton/dev/myumberwm/target/release/deps/liblog-1997b54758139064.rlib" "/home/dalton/dev/myumberwm/target/release/deps/libcfg_if-e0361196973a0e81.rlib" "/home/dalton/dev/myumberwm/target/release/deps/liblibc-a3e882849b8a9f37.rlib" "-Wl,--start-group" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-25c6acf8063a3802.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-539f13c9442f1597.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-37db28e905edb56b.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-9ef2480568df55af.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-1e0f0992cdbecd66.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-6c8e02b8fedc1e5f.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-807e5ad203594490.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-083fce1bea11612a.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-1af568081add9042.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-1395b54a3b3f45bf.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-caba820045f178d5.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-512eb53291f6de7e.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-5efacc5025f9f3d8.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-9c4002b5f79ba0e1.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-90996f4879673567.rlib" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-9ea09a899c3eda46.rlib" "-Wl,--end-group" "/home/dalton/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-ef2408da76957905.rlib" "-Wl,-Bdynamic" "-lxcb" "-lxcb" "-lxcb" "-lxcb" "-lxcb-randr" "-lxcb-render" "-lutil" "-ldl" "-lutil" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc"
  = note: /usr/bin/ld: cannot find -lxcb-randr
          /usr/bin/ld: cannot find -lxcb-render
          collect2: error: ld returned 1 exit status
```

--- a/README.md
+++ b/README.md
@@ -6,31 +6,33 @@ Minimalistic X window manager based on tinywm, inspired by qtile.
 
 ![UmberWM Screenshot](screenshot.jpg)
 
-video introduction [on LBRY](https://open.lbry.com/@goo:c/umberwm:e?r=FKWhS2Vay3CVr66qMZD98HdsLQ2LN7za) or [on youtube](https://youtu.be/5XdFNEq69N0).
+Video introduction [on LBRY](https://open.lbry.com/@goo:c/umberwm:e?r=FKWhS2Vay3CVr66qMZD98HdsLQ2LN7za) or [on youtube](https://youtu.be/5XdFNEq69N0).
 
-# prerequisites
+# Prerequisites
 
-you should have xmodmap installed
+You should have xmodmap installed.
 
-# design goals and features
+You will need XCB bindings with the randr extension. On ubuntu you can install this with `sudo apt install libxcb-randr0-dev`.
 
-  - kiss: only window management (no taskbar, system tray, ...), complex stuff should be done using other programs (rofi, ...)
-  - configuration as code (like qtile, dwm, xmonad)
-  - tiled by default (Binary space partitioning)
-  - supports workspaces
-  - supports multiple displays
-  - single file (~600 LoC)
+# Design goals and features
 
-# using it
+  - Kiss: only window management (no taskbar, system tray, ...), complex stuff should be done using other programs (rofi, ...)
+  - Configuration as code (like qtile, dwm, xmonad)
+  - Tiled by default (Binary space partitioning)
+  - Supports workspaces
+  - Supports multiple displays
+  - Single file (~600 LoC)
 
-umberwm is used/configured in rust, here is how to use it:
+# Using it
 
-1. install rust and cargo https://doc.rust-lang.org/cargo/getting-started/installation.html
-2. clone template project (__:warning: it is a different repository__): `git clone https://github.com/yazgoo/myumberwm`
-3. edit src/main.rs (see comments for more details)
-4. run `cargo build`, binary is available in target/debug/myumerwm
+Umberwm is used/configured in rust, here is how to use it:
 
-add the following to your .xinitrc :
+1. Install rust and cargo https://doc.rust-lang.org/cargo/getting-started/installation.html
+2. Clone template project (__:warning: it is a different repository__): `git clone https://github.com/yazgoo/myumberwm`
+3. Edit src/main.rs (see comments for more details)
+4. Run `cargo build`, binary is available in target/debug/myumerwm
+
+Add the following to your .xinitrc :
 
 ```shell
 exec /path/to/myumberwm mod4

